docs/scos/dev/feature-walkthroughs/202204.0/prices-feature-walkthrough/priceproduct-module-details-reference-information.md

```diff
@@ -63,7 +63,33 @@ class PriceProductConfig extends SprykerPriceProductConfig
 }
 ```
 
-or run `console price-product-store:optimize` from time to time when needed.
+Alternatively, automatic removal of "orphaned records" can be enabled by using `AclEntityOrphanPriceProductStoreRemovalVoterPlugin` in `src/Pyz/Zed/PriceProduct/PriceProductDependencyProvider.php`:
```

Comment:
> enabled

but plugin description is 
>      * - Disables removal of orphaned price product store records on save if `AclEntity` behavior is enabled.

that's confusing

Reply:
Fixed this by adjusting the plugin API specification